### PR TITLE
Remove sra_download workflow in favour of using nf-core/ngsfetch

### PR DESCRIPTION
**Pull request by drpatelh**
Includes some docs updates and minor fixes

### CODE REVIEW

File: docs/usage.md [hunk 1]
```diff
@@ -54,49 +56,131 @@ sample,barcode
...
+    --platform nanopore \
+    --genome 'MN908947.3' \
+    --primer_set_version 3 \
+    --fastq_dir fastq_pass/ \
+    --artic_minion_caller medaka \
+    --artic_minion_medaka_model ./r941_min_high_g360_model.hdf5 \
+    -profile <docker/singularity/podman/conda/institute>
+```
+
+## Illumina primer sets
```
Comment (JoseEspinosa) [inline docs/usage.md:127]:
```suggestion
### Illumina primer sets
```

**Review by JoseEspinosa — APPROVED** [review]
LGTM! 🚀 Just added a comment that probably has a reason to be maintained as it is

File: docs/usage.md [hunk 2]
```diff
@@ -54,49 +56,131 @@ sample,barcode
 | `sample`  | Custom sample name, one per barcode.                                                                                        |
 | `barcode` | Barcode identifier attributed to that sample during multiplexing. Must be an integer.                                       |
 
-## Direct download of public repository data
+> **NB:** Dashes (`-`) in sample names are converted to underscores (`_`) when running [QUAST](http://quast.sourceforge.net/quast) and this causes issues when creating the summary metrics for the pipeline. As a result, dashes in sample names will automatically be replaced with underscores to bypass this issue.
```
[the remark this answers is not in the document]
Comment (drpatelh) [reply docs/usage.md]: Done https://github.com/nf-core/viralrecon/pull/199/commits/3035d3228c61e17c2eba703393ca1f92d96646fd

File: docs/usage.md (hunk 1 above)
Comment (drpatelh) [reply docs/usage.md:127]: This looks fine?

**Comment by drpatelh** [comment]
Thanks @JoseEspinosa ! Another fresh PR incoming so will merge this one so it can be reviewed separately.

**Pull request merged**